tools/validate_configs/golden_copies/configs/igc_tf.yaml

```diff
@@ -41,6 +41,6 @@ deployment_groups:
     source: modules/file-system/filestore
     use: [network0] # wires network_id
```

Comment:
nit: s/network0/network

Reply:
Out of scope . Don't want to make this large even bigger for no good reason.